### PR TITLE
ExPlat: Self-Contained Client Base Package

### PR DESCRIPTION
**Pull request by jessie-ross**
#### Changes proposed in this Pull Request

* This PR adds the base package for the ExPlat self-contained package.

Part of a series rebuilding the reference WIP PR #48475 

#### Testing instructions

* `yarn build` should run successfully

### CODE REVIEW

File: packages/explat-client/.eslintrc.js [hunk 1]
```diff
@@ -0,0 +1 @@
+module.exports = {};
```
Comment (tyxla) [inline packages/explat-client/.eslintrc.js]: Feel free to remove that one if we won't be declaring any special `eslint` rules or exceptions here.

**Review by tyxla — APPROVED** [review]
Looks good to me as a placeholder to build on top of 👍

**Review by yanirs — APPROVED** [review]
It'd be good to also add a readme to this PR since people who see this won't necessarily know what ExPlat is. Just a title, a one-liner description, and simple build instructions would be a good base for later PRs.

**Comment by jessie-ross** [comment]
> It'd be good to also add a readme to this PR since people who see this won't necessarily know what ExPlat is. Just a title, a one-liner description, and simple build instructions would be a good base for later PRs.

Sure I'll add one in, but I was thinking of merging them all together at the end.

**Comment by jessie-ross** [comment]
The desktop and mobile e2e seem to generally fail and are generally ignored.
I will be keeping an eye on their output however to see if there is anything suspicious.

**Pull request merged**